backends/arm/test/models/test_nn_functional.py

```diff
@@ -104,7 +104,15 @@ def test_nn_functional_MI(test_data):
             raise e
 
 
-@parametrize("test_data", module_tests)
+x_fails = {
+    "normalize": "MLETORCH-852: Support aten.index_put.default",
```

Comment:
MLETORCH-* - are these issues tracking ET work? if yes, would it be possible to put them under ET GH? Just for visibility. If I can access these from outside Arm that would be OK too.

Reply:
Hi, 

Thanks for the review. 
We had some discussions yesterday on the best way to do this and we will start looking into this as soon as possible. 

Thanks, 
Saoirse